test/avocado/run-tests

```diff
@@ -69,11 +72,14 @@ def tap_output(machine):
     print("\nTAP output -------------------------")
     print("1..%s" % len(json_info['tests']))
     counter = 1
+    counter_failed = 0
     for t in json_info['tests']:
         test_status = t['status']
-        test_name = os.path.basename(t['id'])
+        test_name_tmp = os.path.basename(t['id']).split(":", 1)
+        test_name = f"{test_name_tmp[0]} ({test_name_tmp[1]})"
```

Comment:
First usage of f-strings here, which means it requires a recent Fedora and RHEL 8. Are you sure that you have a new enough Python (3.6, I believe) available downstream for RHEL testing?

Reply:
yep, sure I'm using python 3.6 collections for RHEL7 and I think  that python 3.6 is default in RHEL8. but I can change this to older syntax

Reply:
It's fine to keep, I just wanted to make sure that this isn't an issue for your RHEL 7 downstream testing.